packages/apps/shopify-app-remix/src/server/authenticate/admin/scope/types.ts

```diff
@@ -1,14 +1,166 @@
+/**
+ * The Scopes API enables embedded apps and extensions to request merchant consent for access scopes.
+ */
 export interface ScopesApiContext {
+  /**
+   * Queries Shopify for the scopes for this app on this shop
+   *
+   * @returns {ScopesDetail} The scope details.
+   *
+   * @example
+   * <caption>Query for scopes details.</caption>
+   * <description>Call `scopes.query` to get scope details.</description>
+   * ```ts
+   * // /app/routes/app.query.tsx
+   *import type { LoaderFunctionArgs } from "@remix-run/node";
+   *import { json } from "@remix-run/node";
+   *
+   *import { authenticate } from "../shopify.server";
+   *
+   *export async function loader({
+   *  request,
+   *}: LoaderFunctionArgs) {
+   *  const { scopes } = await authenticate.admin(request);
+   *
+   *  const response = await scopes.query();
+   *  return json({scopesDetail : response});
```

Comment:
idea: we can reinforce that `ScopesDetail` is what is returned here

```suggestion
   *  const scopesDetail = await scopes.query();
   *  return json({scopesDetail});
```